src/System.Runtime/tests/DBNullTests.InvalidCast.cs

```diff
@@ -0,0 +1,50 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Tests
+{
+    public static partial class DBNullTests
```

Comment:
Is this just a temporarily issue? If so why not just add Skip on them and file and issue as well?